devito/propagator.py

```diff
@@ -1,15 +1,20 @@
-from devito.function_manager import FunctionManager, FunctionDescriptor
+from devito.function_manager import FunctionManager
 from devito.compiler import get_tmp_dir, get_compiler_from_env
 from devito.compiler import jit_compile_and_load, IntelMICCompiler
 import cgen_wrapper as cgen
 from codeprinter import ccode
 import numpy as np
 from sympy import symbols, IndexedBase, Indexed
+from function_manager import FunctionDescriptor
 from collections import Iterable
-from os import path
+from at_controller import *
+from os import getenv, path
 from random import randint
 from hashlib import sha1
 
+_ENV_VAR_TUNE_RANGE = "TUNE_RANGE"
+_ENV_VAR_ISAT_PATH = "ISAT_PATH"
+
```

Comment:
If the choice of whether to autotune or not is passed as a python flag, can't these two be added in the same place?